src/Postgres/src/Eventuous.Postgresql/Subscriptions/PostgresCheckpointStore.cs

```diff
@@ -12,14 +12,17 @@ namespace Eventuous.Postgresql.Subscriptions;
 
 public class PostgresCheckpointStoreOptions {
     // ReSharper disable once ConvertToPrimaryConstructor
-    public PostgresCheckpointStoreOptions(string schema = Postgresql.Schema.DefaultSchema)
+    public PostgresCheckpointStoreOptions(string schema)
         => Schema = schema;
 
+    public PostgresCheckpointStoreOptions() {
+    }
+
     /// <summary>
     /// Override the default schema name.
     /// The property is mutable to allow using ASP.NET Core configuration.
     /// </summary>
-    public string Schema { get; set; }
+    public string Schema { get; set; } = Postgresql.Schema.DefaultSchema;
```

Comment:
could move this defaulting into the default ctor (i.e have it delegate to the other one)